components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam4_resus_mom_soag_tag_uc6.in

```diff
@@ -0,0 +1,280 @@
+Comments
```

Comment:
What's this new mechanism for?

Reply:
This is for test only and will be deleted.